src/Apps/Artwork/Utils/lotIsClosed.ts

```diff
@@ -1,9 +1,26 @@
-export const lotIsClosed = (sale, saleArtwork): boolean => {
-  return (
-    !sale ||
-    !saleArtwork ||
-    sale?.is_closed ||
-    sale?.isClosed ||
-    !!saleArtwork?.endedAt
-  )
+interface LegacySale {
```

Comment:
My attempt at adding typing to this helper (previously `any`) so that you'll know if a fragment where this helper is being used needs to query for `isClosed` or `endedAt` - revealed a couple of places we weren't!